scripts/client/rustdoc.mjs

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env zx
+import 'zx/globals';
+import { cliArguments, workingDirectory } from '../utils.mjs';
+
+const args = cliArguments();
+const manifestPath = path.join(
+  workingDirectory,
+  'clients',
+  'rust',
+  'Cargo.toml'
+);
+
+await $`RUSTDOCFLAGS="--cfg docsrs -D warnings" cargo doc --manifest-path ${manifestPath} --all-features --no-deps ${args}`;
```

Comment:
I could be missing some context here, but `cargo test` (_and_ `test-sbf`) will test-compile your rust docs. It's just `clippy` that doesn't. `fmt` will also make sure your docs/comments are formatted according to the standard. So I think we're pretty covered on rustdocs.

Reply:
`cargo test` and `cargo fmt` don't cover everything that cargo doc covers, such as bad links in doc comments

Reply:
Seems gratuitous to me, but if you guys want the extra coverage I won't object.

Reply:
It tends to catch errors quite often